larpandoracontent/LArControlFlow/StitchingCosmicRayMergingTool.cc

```diff
@@ -524,6 +524,49 @@ void StitchingCosmicRayMergingTool::StitchPfos(const MasterAlgorithm *const pAlg
         for (const ParticleFlowObject *const pPfo : pfoList) pfoVector.push_back(pPfo);
         std::sort(pfoVector.begin(), pfoVector.end(), LArPfoHelper::SortByNHits);
 
+        std::pair<const LArTPC*, const LArTPC*> stitchedLArTPCs;
+
+        try
```

Comment:
Suggest take this entire block and make it into a private member function, which takes a non-const reference to the LArTPCPair and sets the values in the pair.